packages/sdks/snippets/react/src/components/CustomColumns.tsx

```diff
@@ -0,0 +1,39 @@
+import { Blocks, BuilderBlock } from '@builder.io/sdk-react';
+
+interface CustomColumnsProps {
+  columns: { blocks: BuilderBlock[] | undefined }[];
+  builderBlock: { id: string | undefined };
+}
+
+const CustomColumns = (props: CustomColumnsProps) => {
+  return (
+    <div
+      className="two-columns"
```

Comment:
all of these `className`s that you have in custom components are unnecessary and should be removed.

Reply:
this className is still here
```suggestion
```